### PR TITLE
Nukes meteor gamemode

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do

Removes meteor gamemode?

**BUT WHY AFFECTED WHY**

Well, several reasons, but the main 2 are:
- Its just a survival game dictated by how good your ability to survive RNG is
- Performance is absolute ass

image
image

**b-but, those numb-bers arent bad**

Keep in mind, this is with 2.5x **lower** players than highpop, which can run better than this in a lot of cases, and I will gladly cite metrics to prove so.

Its not in rotation for a reason, and that reason is because it sucks. Every time someone mistakenly admin forces it, it is always complained at by both people inside and out of the staff team.

There is a reason that in the time between 2018-03-24 and 2021-05-29, (1142 days), meteor has been ran a whopping **12** times.

Its bad and needs to go.

And for the inevitable "IMPROVE DONT REMOVE" comments. Consider this:
Have you actually played a meteor round? No? Get a grip and stop whining. Change is change.

## Why It's Good For The Game
See above

## Changelog
:cl: AffectedArc07
del: Removed meteor as a gamemode. Its not even in rotation so basically nothing changes.
/:cl:

### CODE REVIEW

File: config/example/config.toml [hunk 1]
```diff
@@ -232,7 +232,6 @@ gamemode_probabilities = [
 	{gamemode = "extend-a-traitormongous", probability = 2}, # Autotraitor
 	{gamemode = "extended", probability = 3},
 	{gamemode = "heist", probability = 0},
```
Comment (AffectedArc07) [inline config/example/config.toml:234]: This is next

Comment (hal9000PR) [reply config/example/config.toml:234]: o7

**Comment by Aurorablade** [comment]
Its not in rotation but can still be used and most admins normally agree upon the event before hand. So I do not feel this is needed.
Performance yes can be bad but normally admins weight it in. The rng survival is a welcome change of pace. My apologies to the people who go I ded pls nerf ong I am going to be irrationally upset at the server host. I am sorry for a but of seethe in my opinion 

In all honesty I feel the people who run this has a plan and normally support it as fast as they can for an event but things go wrong or can go slow to react or it's like Irl where my customers perceptions of time extend 2 minutes into an hour. 

I realize this upset you affected and I know you are on the tail end of the impact. I would hug you if I could. The performance can be hit by more then this at a hairpin and its a constant source ot pain. This is all my opinion though and can be wrong

**Comment by Gatchapod** [comment]
The mode is basically "become an engineer or bridge hobo for the entire round" when you get to the very bottom of its design.
It's like a crappy equivalent of Build Your Own Base event on Lavaland.

On a related note, how feasible it would be to make BYOB an actual game mode so admins don't have to organize everything manually when they want to run it? I'm pretty sure BYOB rounds are universally loved.

**Comment by shmabuu** [comment]
> In all honesty I feel the people who run this has a plan and normally support it as fast as they can for an event but things go wrong

The few times I've seen it run, the individual running the gamemode do support the crew, yes. However, when the people bothering to fix the holes the meteors make are hit by meteors and the rest of the crew have either cryo'd or are just doing nothing, the station gets pelted by meteors until someone throws a tunguska at the station and the shuttle is called.

I had forgotten that this gamemode existed until I saw this. It reminded me of the terrible rounds that it made. A majority of the crew cryo's or doesn't do anything for one main reason -- **performance issues.** 

When the server is slogging as much as it does during meteors, does anyone really want to bother fixing them? I'd rather just move on to a round where the mode is not meteors, personally. Additionally, it's luck if you survive while trying to fix something.

I don't think it'll be missed by many if that. Thank you.

**Comment by Sikroth** [comment]
> The mode is basically "become an engineer or bridge hobo for the entire round" when you get to the very bottom of its design. It's like a crappy equivalent of Build Your Own Base event on Lavaland.
> 
> On a related note, how feasible it would be to make BYOB an actual game mode so admins don't have to organize everything manually when they want to run it? I'm pretty sure BYOB rounds are universally loved.

Unfortunately I don't think there's such a thing as a universally loved gamemode. I know a number of people will cryo or just not deal at all with BYOB rounds. A "become an engineer or hobo in a cramped space for the entire round" gamemode is only slightly less bad in terms of forcing stuff onto people than meteors is, and at least meteors rounds are short.

Not to say there's no merit to making them easier to run to some degree if that many people seem to enjoy it, for sure, but  personally I don't think it should be a fully fledged gamemode either.

**Comment by Denthamos** [comment]
Honestly the meteor game mode is not even needed considering admins can simply force it through extended with nothing but meteor events.

**Review by dearmochi — APPROVED** [review]
Bye

**Pull request merged**